src/transformers/models/gpt_neox/modeling_gpt_neox.py

```diff
@@ -64,6 +65,9 @@ class GPTNeoXPreTrainedModel(PreTrainedModel):
     _no_split_modules = ["GPTNeoXLayer"]
     _skip_keys_device_placement = "past_key_values"
     _supports_flash_attn_2 = True
+    _supports_cache_class = True
+    _supports_quantized_cache = True
+    _supports_static_cache = True
```

Comment:
cool, this is automatically tested, but manually testing would be nice to make sure we have correct outputs!

Reply:
You mean add slow integration tests for each model and cache type, or simply run some generations on my end to see that it makes sense?